server/routes/interview.js

```diff
@@ -0,0 +1,27 @@
+let express = require('express');
+let router = express.Router();
+
+let auth = require('../middleware/authenticateJWT');
+
+// Connect to Interview controller
+let interviewController = require('../controllers/interview');
```

Comment:
Whenever possible, it's preferred to use const instead of let

Reply:
I will have that in mind in future commits, the change has already been made.